tests/test_motor.py

```diff
@@ -26,3 +27,17 @@ def test_motor_soft_limits():
         mot.amove(10)
 
     mot.set_current_as_zero()
+
+def test_motor_where():
+    ctrl = DummyCtrl()
+    mot = Motor('DummyMotor', '1', 0, ctrl)
+    mot.amove(10)
+    mot.where()
+
+def test_motor_scan():
```

Comment:
Purpose of the tests is missing here too.

Reply:
Comment and docstring added to explain the purpose of the added tests.